bin/miniver

```diff
@@ -192,6 +201,15 @@ def install(args):
     print("\n".join((msg, _setup_template)).format(package_dir=package_dir))
 
 
+def ver(args):
+    package_dir = args.package_directory
+    version_location = glob.glob(f"{package_dir}/**/_version.py")[0]
```

Comment:
Testing this out, it seems that the glob returns empty if `_version.py` is directly inside `package_dir`.

It seems that adding [`recursive=True`](https://docs.python.org/3/library/glob.html) will fix this.
We can also catch the cases when `_version.py` does not exist / more than 1 is found:

```suggestion
    try:
        version_location, = glob.glob(f"{package_dir}/**/_version.py", recursive=True)
    except ValueError as err:
        if "not enough" in str(err):
            print(f"'_version.py' not found in '{package_dir}'", file=sys.stderr)
            sys.exit(1)
        elif "too many" in str(err):
            print(f"More than 1 '_version.py' not found in '{package_dir}'",  file=sys.stderr)
            sys.exit(1)
        else:
            raise
```

Reply:
added